docs/discover/get-started-discover.asciidoc

```diff
@@ -0,0 +1,356 @@
+
+The content is copied to your clipboard in the selected format.
+Fields that are not currently added to the table are ignored.
+
+[float]
+[[look-inside-a-document]]
+==== Explore individual result or document details in depth
+
+[[document-explorer-expand-documents]]
+Dive into an individual document to view its fields and the documents
+that occurred before and after it.
+
+. In the document table, click the expand icon
+image:images/expand-icon-2.png[double arrow icon to open a flyout with the document details]
+to show document details.
++
+[role="screenshot"]
+image:images/document-table-expanded.png[Table view with document expanded]
+
+. Scan through the fields and their values. You can filter the table in several ways:
+** If you find a field of interest,
+hover your mouse over the *Field* or *Value* columns for filters and additional options. 
+** Use the search above the table to filter for specific fields or values, or filter by field type using the options to the right of the search field.
+** You can pin some fields by clicking the left column to keep them displayed even if you filter the table.
++
+TIP: You can restrict the fields listed in the detailed view to just the fields that you explicitly added to the **Discover** table, using the **Selected only** toggle. In ES|QL mode, you also have an option to hide fields with null values.
+
+. To navigate to a view of the document that you can bookmark and share, select ** View single document**.
+
+. To view documents that occurred before or after the event you are looking at, select
+**View surrounding documents**.
+
+
+
+
+[float]
+[[search-in-discover]]
+=== Search and filter data
+
+[float]
+==== Default mode: Search and filter using KQL
+
+One of the unique capabilities of **Discover** is the ability to combine
+free text search with filtering based on structured data.
+To search all fields, enter a simple string in the query bar.
+
+[role="screenshot"]
+image:images/discover-search-field.png[Search field in Discover]
+
+To search particular fields and
+build more complex queries, use the <<kuery-query,Kibana Query language>>.
+As you type, KQL prompts you with the fields you can search and the operators
+you can use to build a structured query.
+
+For example, search the ecommerce sample data for documents where the country matches US:
+
+. Enter `g`, and then select *geoip.country_iso_code*.
+. Select *:* for equals, and *US* for the value, and then click the refresh button or press the Enter key.
+. For a more complex search, try:
++
+```ts
+geoip.country_iso_code : US and products.taxless_price >= 75
+```
+
+[[filter-in-discover]]
+Whereas the query defines the set of documents you are interested in,
+filters enable you to zero in on subsets of those documents.
+You can filter results to include or exclude specific fields, filter for a value in a range,
+and more.
```

Comment:
The query bar allows to use KQL or Lucene language for filtering and the filters provide a "clickable" way to achieve mostly the same (plus a custom QueryDSL). With both ways user can include or exclude.

Reply:
I see, I didn't update that part from the previous version, but I agree it doesn't read well. I'll try to rephrase it to represent more accurately what you're describing. Let me know what you think, thanks!
```suggestion
With the query field, you can filter data using the KQL or Lucene languages. You can also use the **Add filter** function available next to the query field to build your filters manually or edit them as Query DSL.
```

Reply:
> to build your filters manually or edit them as Query DSL

Or "to build your filters one by one or define them as Query DSL"

Reply:
> With the query field

With the query input

Reply:
Applied 👍 thanks